src/Infrastructure/Value/Translation/TranslatableMessage.php

```diff
@@ -0,0 +1,83 @@
+<?php declare(strict_types=1);
+
+namespace PhpTuf\ComposerStager\Infrastructure\Value\Translation;
+
+use PhpTuf\ComposerStager\Domain\Exception\InvalidArgumentException;
+use PhpTuf\ComposerStager\Domain\Factory\Translation\TranslatableAwareTrait;
+use PhpTuf\ComposerStager\Domain\Service\Translation\TranslatorInterface;
+use PhpTuf\ComposerStager\Domain\Translation\TranslatableFactoryInterface;
+use PhpTuf\ComposerStager\Domain\Value\Translation\TranslatableInterface;
+use PhpTuf\ComposerStager\Domain\Value\Translation\TranslationParametersInterface;
+use Throwable;
+
+/**
+ * Handles a translatable message.
+ *
+ * @see \PhpTuf\ComposerStager\Domain\Translation\TranslatableFactoryInterface
+ * @see https://symfony.com/doc/current/translation.html
+ *
+ * @package Translation
+ *
+ * @api This class may be instantiated directly or created using the path factory above.
+ */
+final class TranslatableMessage implements TranslatableInterface
+{
+    use TranslatableAwareTrait;
+
+    /**
+     * Creates a translatable message.
+     *
+     * @param string $message
+     *   A message containing optional placeholders corresponding to parameters (next). Example:
+     *   ```php
+     *   $message = 'Email %name at <a href="mailto:%email">%email</a>.';
```

Comment:
Within Composer Stager, all translatable messages are text, not HTML. Therefore, let's not use HTML in this example. Let's instead base an example on some message that exists in Composer Stager. +1 though to showing an example message with at least 2 placeholders.

One of the reasons I want to avoid HTML in this example is that in Drupal, you would never use a placeholder that starts with % inside of an HTML attribute value. We have other prefixes for that, but there's no need to complicate Composer Stager by even mentioning those other possible prefixes since there's no use case for them within Composer Stager messages.